src/components/Account/AccountTransaction/AccountTxPanel.scss

```diff
@@ -0,0 +1,53 @@
+.accountPanel {
+  .content {
+    padding: 0 24px 32px;
+  }
+
+  .inputWrapper {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    align-items: center;
+  }
+
+  .smallInputChild{
+    width: 22vh;
+
+    border-radius: 4px 0 0 4px !important;
+    border-right: 10px !important;
+  }
+
+  .selectChild {
+    margin-right: 10px;
+    align-self: flex-end;
+    height: 36.5px;
+    border: 2px solid #dde2ec;
+
+    border-radius: 0 4px 4px 0 !important;
+    border-left: 10px !important;
+
+    &::placeholder {
+      color: #999;
+    }
+
+    &:focus {
+      outline: 0;
+      color: #495057;
+      border-color: #047bf8;
+    }
+  }
+
+  .smallInputChild:focus > .selectChild {
```

Comment:
This currently does not work - I'd like to have the border of the selectChild to be highlighted when smallInputChild is focused and the other way around. This would create the feeling of 1 large merged input w/ select

Reply:
```scss
.smallInputChild:focus {
  .selectChild {
     // Do your thing
  }
}
```